### PR TITLE
Add Sonoff S31, iFan02, 4chProR2 and RF Bridge

### PR DESCRIPTION
**Pull request by rgabrielson11**

### CODE REVIEW

File: _devices/sonoff-rf-bridge.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+      code: '001111111110111100100111'
+      protocol: 1     
+    on_press:
+      - binary_sensor.template.publish:
+          id: Sensor2
+          state: OFF    
+  - platform: template
+    name: "Sensor2 State"
+    device_class: window
+    id: Sensor2
```
Comment (jonathanadams) [inline _devices/sonoff-rf-bridge.md]: Should this be part of a `sensor:` component?

**Review by jonathanadams — CHANGES_REQUESTED** [review]

**Comment by jonathanadams** [comment]
I have made some minor changes to file names and corrected the closing tags for the code block in some files.

In regards to the Sonoff RF Bridge I have requested a change as I'm not familiar with this device.

Once changes are made I will merge. Thanks

**Comment by rgabrielson11** [comment]
Added 'binary sensor'.  thanks for catching this

File: _devices/sonoff-rf-bridge.md (hunk 1 above)
Comment (rgabrielson11) [reply _devices/sonoff-rf-bridge.md]: Yes, added binary sensor.  missed this.

**Pull request merged**